doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -989,10 +995,14 @@ Example of SSH logging:
     "client": {
         "proto_version": "2.0",
         "software_version": "OpenSSH_6.7",
+        "ssh.hassh": "ec7378c1a92f5a8dde7e8b7a1ddf33d1",
+        "ssh.hassh.string": "curve25519-sha256,diffie-hellman-group14-sha256,diffie-hellman-group14-sha1,ext-info-c",
      },
     "server": {
         "proto_version": "2.0",
         "software_version": "OpenSSH_6.7",
+        "ssh.hassh.server": "ec7378c1a92f5a8dde7e8b7a1ddf33d1",
```

Comment:
Maybe I do not remember the discussion.
But it still feels strange/redundant to have `ssh.hassh.server` in `server` block

Can we have only `ssh.hassh` and `ssh.hassh.string` and then use the direction ?